e2e-mobile-tests/tests/locators/SignUpLocators.ts

```diff
@@ -1,5 +1,185 @@
 const signUpLocators = {
+  /* =====================================================
+        CREATE MASTER PASSWORD
+  ===================================================== */
 
+  createPasswordScreenAndroid: '~create_password_screen',
+  createPasswordScreeniOS: '~create_password_screen',
```

Comment:
I am not an expert in e2e mobile tests, but seems like a lot of duplication here, is there any optimizations to do not add iOS and Android ? if it's the same value and only use it if it's different